config.yaml

```diff
@@ -19,7 +19,8 @@ submodels:
 
     - name: ocean
       model: mom
-      ncpus: 228
+      #ncpus: 228
```

Comment:
If this is a permanent change will want to remove this commented line

Reply:
Also should remove the commented walltime option and the project also.

Reply:
ta, done